File: sample/dotnet/BestsellerApi.ApiService/Models/TrackingInfo.cs
namespace BestsellerApi.ApiService.Models;

public record TrackingInfo(
    string TrackingNo,
    string Status,
    string CurrentLocation,
    DateTimeOffset? EstimatedDelivery,
    DateTimeOffset? DeliveryDate,
    IReadOnlyList<TrackingEvent>? History
);

File: sample/java/src/main/java/com/bestseller/api/model/TrackingInfo.java
package com.bestseller.api.model;

import java.time.Instant;
import java.util.List;

public record TrackingInfo(
    String trackingNo,
    String status,
    String currentLocation,
    Instant estimatedDelivery,
    Instant deliveryDate,
    List<TrackingEvent> history
) {
}
